### ISSUE TITLE
Inability to create a new project from an open project

### ISSUE THREAD

**Issue by jake-stack** [saga-vcs/saga-excel#36]
When you're inside a project and you wan to make a new project so you click "create saga project" , the sidebar does not show the options to create a new project

**Comment by naterush** [comment]
@jake-stack thanks for the report. This is when there is already a saga project created, it will open to the last taskpane state. 

We should improve the language, and possibly think about not having "create" be the permanent button but rather "open taskpane" or something...

**Comment by jake-stack** [comment]
Agreed on the language simplifying but it would also be nice to have the
ability to open a new project that perhaps opens in a new window (as
opposed to just switching you to that project), with the click of a button
